### PR TITLE
Fixing docgen issue

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -181,6 +181,9 @@
       "block-no-empty": null
     }
   },
+  "resolutions": {
+    "ast-types": "^0.14.0"
+  },
```
Comment (vishalnarkhede) [inline package.json:186]: https://github.com/reactjs/react-docgen/issues/463#issuecomment-698769524

**Review by jaapbakker88 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mahboubii) [reply package.json:186]: nice

**Review by mahboubii — APPROVED** [review]
🌮

File: src/components/Chat/Chat.js [hunk 2]
```diff
@@ -52,16 +52,22 @@ const Chat = ({
...
 
   useEffect(() => {
-    const userAgent = client.getUserAgent();
-    if (!userAgent.includes('stream-chat-react')) {
-      // should result in something like:
-      // 'stream-chat-react-2.3.2-stream-chat-javascript-client-browser-2.2.2'
-      client.setUserAgent(`stream-chat-react-${version}-${userAgent}`);
+    const setUserAgent = () => {
+      const userAgent = client.getUserAgent();
+      if (!userAgent.includes('stream-chat-react')) {
```
Comment (virdesai) [inline src/components/Chat/Chat.js:57]: could this not be `stream-chat-react-native` too?

**Review by virdesai — APPROVED** [review]

File: src/components/Chat/Chat.js [hunk 3]
```diff
@@ -52,16 +52,22 @@ const Chat = ({
...
+        client.setUserAgent(`stream-chat-react-${version}-${userAgent}`);
+      }
+    };
+
+    if (client) {
+      setUserAgent();
     }
     // don't want client in dep array because it is a required
     // prop for this component and we only want this run on mount
     // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (vicnicius) [inline src/components/Chat/Chat.js:69]:
```suggestion
```

**Pull request merged**